group_vars/all/logrotate.yml

```diff
@@ -17,4 +17,4 @@ logrotate_scripts:
         if [ -d /etc/logrotate.d/httpd-prerotate ]; then \
               run-parts /etc/logrotate.d/httpd-prerotate; \
             fi \
-      postrotate: invoke-rc.d nginx rotate >/dev/null 2>&1
+      postrotate: "[ -f /var/run/nginx.pid ] && kill -USR1 `cat /var/run/nginx.pid`"
```

Comment:
I'll test it out. I had seen a minor reference to it at [Stack Exchange](http://unix.stackexchange.com/questions/209585/nginx-logrotate-error-on-cron-job) when I initially looked at the issue, but I stuck with the `kill` command based-solution I'd used before.

According to [Nginx Essentials](http://shop.oreilly.com/product/9781785289538.do) [ that's a clean link, not an affiliate one ] on p.40, `rotate` sends a USR1 signal, so it should be good.